### ISSUE TITLE
How I can contact you by email?

### ISSUE THREAD

**Issue by Neustradamus** [fingergit/shairplay4w#1]
@fingergit: How I can contact you by email?

Can I have your email address?

You can remove this ticket after sending the reply ^^

Thanks in advance.

**Comment by Neustradamus** [comment]
@fingergit: Happy New Year 2022!

Can you specified me your email address?

Thanks in advance.

**Comment by fingergit** [comment]
@Neustradamus Happy New Year 2022! For any question, you can show here, thank you.

**Comment by Neustradamus** [comment]
@fingergit: Thanks for your wishes.

It is better by email, thanks in advance.

**Comment by fingergit** [comment]
@Neustradamus  Please leave your email here, I will mail to you later. Thank you.

**Comment by Neustradamus** [comment]
@fingergit: username @ hotmail dot com

I remove the comment, you will have an email from github with...

**Comment by Neustradamus** [comment]
@fingergit: username @ hotmail dot com

I remove the comment, you will have an email from github with...

**Comment by Neustradamus** [comment]
@fingergit: I have removed my previous comment with.
I wait you by email, inform me here, when you have sent the email, thanks in advance.

**Comment by fingergit** [comment]
@Neustradamus Sorry, I don't know your email address. Please leave your email address here, then I will contact you by email.

**Comment by Neustradamus** [comment]
@fingergit: For each new ticket or new comment, you do not received an email?

**Comment by fingergit** [comment]
@Neustradamus Every mail was received from "notifications@githum.com".

**Comment by Neustradamus** [comment]
Yes it is, I have noted in a comment and I have removed the comment after, you have received my email in a github email...

**Comment by fingergit** [comment]
@Neustradamus I have sent a mail, did you receive it?

**Comment by Neustradamus** [comment]
@fingergit: Yes I have thanks, it was in spam, I contact you soon! :)

**Comment by Neustradamus** [comment]
@fingergit: Email sent!
